src/share/poudriere/image.sh

```diff
@@ -351,7 +351,6 @@ touch ${WRKDIR}/src.conf
 make -C ${mnt}/usr/src DESTDIR=${WRKDIR}/world BATCH_DELETE_OLD_FILES=yes SRCCONF=${WRKDIR}/src.conf delete-old delete-old-libs
 
 [ ! -d "${EXTRADIR}" ] || cp -fRPp ${EXTRADIR}/ ${WRKDIR}/world/
-mv ${WRKDIR}/world/etc/login.conf.orig ${WRKDIR}/world/etc/login.conf
```

Comment:
Actually it *may* exist if the jail is old enough. We need to keep the `mv` if it exists like in jail.sh:
```
if [ -f "${JAILMNT}/etc/login.conf.orig" ]; then
        mv -f ${JAILMNT}/etc/login.conf.orig ${JAILMNT}/etc/login.conf
```

Reply:
Shouldn't it be:
`mv -f ${JAILMNT}/etc/login.conf.orig ${WRKDIR}/etc/login.conf`
in place of:
`mv -f ${JAILMNT}/etc/login.conf.orig ${JAILMNT}/etc/login.conf`
?

Reply:
Yes you are right.